plugins/org.jkiss.dbeaver.ext.postgresql/OSGI-INF/l10n/bundle_ro_RO.properties

```diff
@@ -0,0 +1,839 @@
+tree.tablechildren_columns.node.name=Coloane Child Tables
+tree.rules.node.name=Reguli
+tree.rules.node.tip=Reguli de tabel\u0103
+tree.rule.node.name=Regul\u0103
+tree.policies.node.name=Politici
+tree.policies.node.tip=Politici RLS
+tree.policy.node.name=Politica
+tree.policy.node.tip=Politica RLS
+tree.schema.node.name=Schem\u0103
+tree.schemas.node.name=Scheme
+tree.schemas.node.tip=Scheme
+tree.sequence.node.name=Secven\u0163\u0103
+tree.sequences.node.name=Secven\u0163e
+tree.sequences.node.tip=Secven\u0163e
+tree.sessions.node.name=Sesiuni
+tree.locks.node.name=Bloc\u0103ri
+tree.locks.node.tip=Manager de bloc\u0103ri a serverului
+tree.table.node.name=Tabel
+tree.tables.node.name=Tabele
+tree.tables.node.tip=Tabele
+tree.tablespace.node.name=Tablespace
+tree.tablespaces.node.name=Spa\u0163ii de tabel\u0103
+tree.trigger.node.name=Trigger
+tree.triggers.node.name=Trigger
+tree.triggers.node.tip=Trigger de tabel\u0103
+tree.event.trigger.node.name=Trigger de eveniment
+tree.event.triggers.node.name=Trigger de evenimente
+tree.event.triggers.node.tip=Trigger de evenimente
+tree.tview.node.name=Vedere
+tree.tviews.node.name=Vizualiz\u0103ri
+tree.tviews.node.tip=Vizualiz\u0103ri
+tree.type.node.name=Tip de date
+tree.types.node.name=Tipuri de date
+tree.types.node.tip=Tipuri de date
+tree.aggregates.node.name=Func\u0163ii agregate
+tree.aggregates.node.tip=Func\u0163ii agregate
+tree.aggregate.node.name=Func\u0163ia agregat\u0103
+tree.role.node.name=Rol
+tree.roles.node.name=Roluri
+tree.roles.node.tip=Roluri \u015Fi utilizatori
+tree.role.member.node.name=Rol
+tree.role.members.node.name=Membrii
+tree.role.members.node.tip=Membrii
+tree.role.belong.node.name=Rol
+tree.role.belongs.node.name=Roluri
+tree.role.belongs.node.tip=Apar\u0163ine
+tree.opClasses.node.name=Clasele de operatori
+tree.opClass.node.name=Clasa de operator
+tree.opFamilies.node.name=Familii de operatori
+tree.opFamily.node.name=Familia de operatori
+tree.jobs.node.name=Locuri de munc\u0103
+tree.jobs.node.description=Lista joburilor programate care urmeaz\u0103 s\u0103 fie executate de pgAgent.
+tree.job.node.name=Job
+tree.job.steps.node.name=Pa\u015Fi
+tree.job.step.node.name=Etapa
+tree.job.schedules.node.name=Planificatoare
+tree.job.schedule.node.name=Planificator
+tree.job.classes.node.name=Clase
+tree.job.class.node.name=Clas\u0103
+tree.foreign.tables.node.name=Tabelele str\u0103ine
+tree.foreign.table.node.name=Tabel str\u0103in\u0103
+tree.foreign.tables.node.tip=Tabelele str\u0103ine
+
+driver.postgresql.description=Driver standard PostgreSQL
+driver.edb.description=Driver standard EDB
+driver.timescale.description=Driver JDBC de timp
+driver.yugabytedb.description=Driver JDBC YugabyteDB
+driver.cockroach.description=Driver JDBC CockroachDB
+driver.gcloudpg.description=Driver Google Cloud SQL PostgreSQL JDBC
+driver.cratedb.description=Driver CrateDB JDBC
+driver.materialize.description=Materializa\u0163i driverul JDBC
```

Comment:
So, I'm confused. Is the word "driver" translatable to the Romanian language or not? You translated it here, but not for other drivers' names.

Reply:
No. In context of DB the word **driver** cannot be translated. I will correct it shortly.

Reply:
Okay

Reply:
Solved in [10f1ea4](https://github.com/dbeaver/dbeaver/pull/22696/commits/10f1ea478192d4bd7ebd5d3c9229a3c08be55b85)